src/Nethermind/Nethermind.Db/DbOnTheRocks.cs

```diff
@@ -40,8 +40,7 @@ public abstract class DbOnTheRocks : IDb, IDbWithSpan
 
         public DbOnTheRocks(string basePath, string dbPath, IDbConfig dbConfig, ILogManager logManager = null) // TODO: check column families
         {
-            var directory = PathUtils.GetExecutingDirectory();
-            var fullPath = Path.Combine(directory, basePath, dbPath);
+            var fullPath = Path.Combine(basePath.GetApplicationResourcePath(), dbPath);
```

Comment:
My intended usage here was dbPath.GetApplicationResourcePath(basePath) but in the end I think it will be same behaviour